app/models/admin/r_and_r_item.rb

```diff
@@ -0,0 +1,100 @@
+class Admin::RAndRItem < ApplicationRecord
+  belongs_to :digitization_queue_item, optional: true
+
+  before_destroy :remove_reference_from_dq_table
+
+  scope :open_status,   -> { where.not(status: "closed_r_and_r_request") }
+  scope :closed_status, -> { where(    status: "closed_r_and_r_request") }
+
+  # for now just validate bib numbers to not have the extra digit.
+  # We could try to 'automatically' fix them if this is still too
+  # confusing for curators
+  class StartsWithBValidator < ActiveModel::EachValidator
+    def validate_each(record, attribute, value)
+      value = record.send(attribute)
+      unless (value.blank? || value.starts_with?("b"))
+        record.errors[attribute] << (options[:message] || "must start with b")
+      end
+    end
+  end
+  validates :bib_number, length: { is: 8 }, allow_blank: true, starts_with_b: true
+
+  # collecting areas could have been normalized as a separate table, but
+  # not really needed, we'll just leave it as a controlled string.
+  COLLECTING_AREAS = %w{archives photographs rare_books modern_library museum_objects museum_fine_art}
+  validates :collecting_area, inclusion: { in: COLLECTING_AREAS }
```

Comment:
use some constant that already exists for collection areas in Admin::DigitizationQueueItem::COLLECTING_AREAS

Reply:
Done. I could also wrap this in a method - not sure exactly if this works.

Reply:
Stay tuned... gonna make an adjustment.

Reply:
Never mind -- the way it is is fine. Was confusing this with STATUSES, which is actually used in a presenter; slightly different scenario.